nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -216,6 +220,30 @@ public static void main(String[] args) throws IOException, InterruptedException
                 dumpFile = null;
                 verbose = false;
             }
+        } else if (cmd.equalsIgnoreCase("status-history")) {
+            if (args.length != 4) {
+                System.err.printf("Wrong number of arguments: %d instead of 4, the command parameters are: " +
+                        "status-history --days <number of days> <dumpFile>%n", args.length);
```

Comment:
It might be my perfonal preference, but the "--days" argument does not provide any information, thus it can be removed. I think, maybe the best would be to remove that and swap the two remaining arguments. By this, we can offer a default for days (1? 2?) and allow users privde the file only

Reply:
I agree to remove the --days option; I will add an optional 1 day default value to it, however I would not change the order because the diagnostic command follows the same order of params.

Reply:
Fine with me, thanks